src/services/course.test.js

```diff
@@ -0,0 +1,275 @@
+import { Op } from 'sequelize';
+import { expect } from '@playwright/test';
+import db, { Course } from '../models';
+import {
+  csvImport,
+} from './course';
+
+describe('Course', () => {
+  beforeAll(async () => {
+
+  });
+
+  afterAll(async () => {
+    await db.sequelize.close();
+  });
+
+  describe('csvImport', () => {
+    const headings = ['course name'];
+    const courseNamesToCleanup = [];
+
+    afterEach(async () => {
+      await Course.destroy({
+        where: {
+          nameLookUp:
+          {
+            [Op.in]: courseNamesToCleanup,
+          },
+        },
+        force: true,
+      });
+    });
+
+    it('creates a new course', async () => {
+      const newCourseName = 'Sample Course Name to Create';
+      courseNamesToCleanup.push(newCourseName.replace(/[^a-zA-Z0-9]/g, '').toLowerCase().trim());
```

Comment:
doesn't super matter but you shouldn't need the trim here since the regex is stripping out spaces already

Reply:
removed.